notebooks/00-intro.ipynb

```diff
@@ -262,8 +262,8 @@
     "    \"z\": 8689.83,\n",
     "    \"y\": 9674.05,\n",
     "}\n",
-    "Rubin = btk.survey.Survey(\n",
-    "    \"Rubin\",\n",
+    "LSST = btk.survey.Survey(\n",
+    "    \"LSST\",\n",
     "    pixel_scale=0.2,\n",
     "    effective_area=32.4,\n",
     "    mirror_diameter=8.36,\n",
```

Comment:
oudated

Reply:
Thanks @aboucaud for pointing out where the docs/notebooks need to be updated. I linked to your comments in #270 since it will be done in a future PR